_docs/principles/accessibility.md

```diff
@@ -121,27 +218,35 @@ Other optional indicators to consider are the following elements:
 
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-### Color and contrast
+## Page layout guidelines
 
-Readability requires sufficient contrast between the text and its background. Low contrast is especially problematic for users with low vision who also often have contrast sensitivity.
+Consider the following page layout guidelines when you construct an accessible UI.
 
-To ensure text is readable, Helix adheres to the accessibility standards defined by [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/). These standards are based on contrast ratios calculated from text size, text color, and background color.
+{% endcolumn %}
 
-Helix minimum contrast standards require the following elements:
+</div>
 
-- Small text should have a contrast ratio of at least 4.5:1 against its background color.
-- Large text (at 14px bold or 18px regular and up) should have a contrast ratio of at least 3:1.
-- We recommend using the contrast ratios for icons or other critical elements.
+</section>
 
-{% endcolumn %}
+<section class="static-section"  markdown="1">
 
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+<div class="hxRow"  markdown="1">
 
-{% figure [caption: "Helix typography styles with their corresponding contrast ratios. All ratios were calculated using a `#fafafa` (`gray 25`) background color."] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-typography.png" width="598"/>
-{% endfigure %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+### Be consistent with headings
+
+**Alternate text**
+
+Screen readers read headings in the order you define them in the document, and not in the order they appear on the screen. For example, a Heading 1 and a Heading 2 can appear in that order on the screen, but you can define them in the document so that Heading 2 is read before Heading 1.
```

Comment:
I mentioned the ordering in my previous review merely to clarify how screen readers work.  This text suggests that you can define a Heading 2 before a Heading 1 (which is possible), but it violates semantics.

Document order is important to screen readers in order to provide a document outline as well as logical landmarks for navigating content. Incorrect heading levels scramble the outline and confuses screen reader users.

Reply:
Done.